rmw_opensplice_cpp/src/rmw_node.cpp

```diff
@@ -206,7 +206,8 @@ rmw_create_node(const char * name, const char * namespace_, size_t domain_id)
   }
   memcpy(const_cast<char *>(node->name), name, strlen(name) + 1);
 
-  node->namespace_ = reinterpret_cast<const char *>(rmw_allocate(sizeof(char) * strlen(namespace_) + 1));
+  node->namespace_ = reinterpret_cast<const char *>(
```

Comment:
the same casting logic has been used in all the namespace implementation PRs (e.g. https://github.com/ros2/rmw_connext/pull/220/files#diff-9bba412d4597100825da2b2307403d08R436). So I'm not going to change it in this one. Feel free to open a set of follow-up PRs to fix the cast logic all at once